addons/point_of_sale/models/res_config_settings.py

```diff
@@ -1,21 +1,304 @@
+    pos_picking_type_id = fields.Many2one(related='pos_config_id.picking_type_id', readonly=False)
+    pos_pricelist_id = fields.Many2one('product.pricelist', string='Default Pricelist', compute='_compute_pos_pricelist_id', readonly=False, store=True, pos='pricelist_id')
+    pos_product_load_background = fields.Boolean(related='pos_config_id.product_load_background', readonly=False)
+    pos_proxy_ip = fields.Char(string='IP Address', compute='_compute_pos_proxy_ip', readonly=False, store=True, pos='proxy_ip')
+    pos_receipt_footer = fields.Text(string='Receipt Footer', compute='_compute_pos_receipt_header_footer', readonly=False, store=True, pos='receipt_footer')
+    pos_receipt_header = fields.Text(string='Receipt Header', compute='_compute_pos_receipt_header_footer', readonly=False, store=True, pos='receipt_header')
+    pos_restrict_price_control = fields.Boolean(related='pos_config_id.restrict_price_control', readonly=False)
+    pos_rounding_method = fields.Many2one(related='pos_config_id.rounding_method', readonly=False)
+    pos_route_id = fields.Many2one(related='pos_config_id.route_id', readonly=False)
+    pos_selectable_categ_ids = fields.Many2many('pos.category', compute='_compute_pos_selectable_categ_ids')
+    pos_sequence_id = fields.Many2one(related='pos_config_id.sequence_id')
+    pos_set_maximum_difference = fields.Boolean(related='pos_config_id.set_maximum_difference', readonly=False)
+    pos_ship_later = fields.Boolean(related='pos_config_id.ship_later', readonly=False)
+    pos_start_category = fields.Boolean(related='pos_config_id.start_category', readonly=False)
+    pos_tax_regime_selection = fields.Boolean(related='pos_config_id.tax_regime_selection', readonly=False)
+    pos_tip_product_id = fields.Many2one('product.product', string='Tip Product', compute='_compute_pos_tip_product_id', readonly=False, store=True, pos='tip_product_id')
+    pos_use_pricelist = fields.Boolean(related='pos_config_id.use_pricelist', readonly=False)
+    pos_warehouse_id = fields.Many2one(related='pos_config_id.warehouse_id', readonly=False, string="Warehouse (PoS)")
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        # STEP: Remove the 'pos' fields from each vals.
+        #   They will be written atomically to `pos_config_id` after the super call.
+        pos_config_id_to_fields_vals_map = {}
+
+        for vals in vals_list:
+            pos_config_id = vals.get('pos_config_id', False)
+            if pos_config_id:
+                pos_fields_vals = {}
+                for _field in self._fields.values():
+                    val = vals.get(_field.name, None)
+
+                    # Add only to pos_fields_vals if
+                    #   1. _field is in vals -- meaning, the _field is in view.
+                    #   2. _field has 'pos' attribute
+                    if hasattr(_field, 'pos') and val is not None:
```

Comment:
Need to check if the attribute is a valid `pos.config` field. But check here is too late. Should be during installation.